setup.rst

```diff
@@ -16,6 +16,7 @@ Installing Symfony
 Before creating your first Symfony application, make sure to meet the following
 requirements:
 
+* Installed :doc:`Symfony client <https://symfony.com/download>`
```

Comment:
Maybe write, optional but recommended?

this tool is not a requirement, just an new option (even if cool/easy etc)